config/overlays/mutation_webhook/kustomization.yaml

```diff
@@ -0,0 +1,22 @@
+# TODO: this is a temporary kustomization for the mutation webhook
+# It is kept separate until the mutation feature is stable enough,
+# when the mutation webhook should be moved to config/webhooks/manifests.yaml  
+
+namespace: gatekeeper-system
+
+bases:
+  - ../dev
+
+resources:
+  - webhook.yaml
+  - mutations.gatekeeper.sh_assign.yaml
+  - mutations.gatekeeper.sh_assignmetadata.yaml
+
+
+patchesJson6902:
+- target:
+    group: rbac.authorization.k8s.io
+    version: v1
+    kind: ClusterRole
+    name: manager-role
+  path: webhook_permissions_patch.yaml
```

Comment:
nit: new line

Reply:
Done
although it conflicts with [this comment](https://github.com/open-policy-agent/gatekeeper/pull/881#discussion_r530655080) :)